src/heretic/sb_bar.c

```diff
@@ -1051,69 +1051,169 @@ void DrawInventoryBar(void)
 
 void DrawFullScreenStuff(void)
 {
-    const char *patch;
+	const char *patch;
```

Comment:
No unrelated white space changes, please.

Reply:
I fixed it.